src/theme/dropdown-menu.ts

```diff
@@ -19,6 +19,15 @@ export default (options: Required<ModuleOptions>) => ({
     itemLabelExternalIcon: 'inline-block size-3 align-top text-[var(--ui-text-dimmed)]'
   },
   variants: {
+    color: {
+      ...Object.fromEntries((options.theme.colors || []).map((color: string) => [color, ''])),
+      neutral: ''
+    },
+    loading: {
```

Comment:
Why move this?

Reply:
I guess I forgot to move it back when I was experimenting with colors. Rolled back.